src/components/availability-panel/drag-drop-context/index.tsx

```diff
@@ -48,8 +48,23 @@ const DragDropContextWrapper: FC<dragDropProps> = ({
   const ref = useRef<string[]>([]);
 
   useEffect(() => {
-    setTaskList(unAssignedTasks);
-    setMemberList(idleMembers);
+    const appendNewTasks = (
+      taskList: task[],
+      unAssignedTasks: task[]
+      ) :task[] => {
+        const oldTasksIds = taskList.map((Task: task) => Task.id);
```

Comment:
Have we defined the interface in smallcase?

Reply:
maybe taskItem or just item, Task looks bit different in my view

Reply:
`oldTasksIds.includes(Task.id)`
Can't we directly do `!taskList.includes(taskItem/Task in this case?)

Reply:
https://github.com/Real-Dev-Squad/website-status/blob/4367c70a1ef222bf9a5a1d163f16c1a902833452/src/interfaces/task.type.ts#L3
@harshith-venkatesh yes, the interface is defined in smallcase.

Reply:
taskItem/Task is an object so can't check for them. So going with id here seems a better way to compare them.

Reply:
https://bobbyhadz.com/blog/javascript-check-if-array-contains-object

Reply:
Could you look into this and see if anything would be helpful to the above code?

Reply:
I have rarely seen someone defining types in lowercase.
https://www.typescriptlang.org/docs/handbook/2/template-literal-types.html

Could this be looked into please
Either now or create a new ticket for the same

Reply:
@harshith-venkatesh we need to filter out new tasks and old tasks here. With .some it will return us the boolean value but not the actual taksItem element. 
I am trying to filter the tasks whose id is not present in oldTasksIds array which becomes the new tasks, so as to append them with the oldTasks.

Reply:
@harshith-venkatesh currently we have all the types in lowercase in `interfaces` folder. Changing them into caps, will require to change in all places where we have used it.

Reply:
Got it , in that case I am ok with the changes and could you create a new issue and have it discussed with @ankushdharkar for the same please